pkg/ottl/ottlfuncs/func_time_test.go

```diff
@@ -0,0 +1,238 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package ottlfuncs
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/ottl"
+)
+
+func Test_Time(t *testing.T) {
+	tests := []struct {
+		name     string
+		time     ottl.StringGetter[interface{}]
+		format   string
+		expected time.Time
+	}{
+		{
+			name: "simple short form",
+			time: &ottl.StandardStringGetter[interface{}]{
+				Getter: func(ctx context.Context, tCtx interface{}) (interface{}, error) {
+					return "2023-04-12", nil
+				},
+			},
+			format:   "%Y-%m-%d",
+			expected: time.Date(2023, 4, 12, 0, 0, 0, 0, time.Local),
+		},
+		{
+			name: "simple short form with short year and slashes",
+			time: &ottl.StandardStringGetter[interface{}]{
+				Getter: func(ctx context.Context, tCtx interface{}) (interface{}, error) {
+					return "11/11/11", nil
+				},
+			},
+			format:   "%d/%m/%y",
+			expected: time.Date(2011, 11, 11, 0, 0, 0, 0, time.Local),
+		},
+		{
+			name: "month day year",
+			time: &ottl.StandardStringGetter[interface{}]{
+				Getter: func(ctx context.Context, tCtx interface{}) (interface{}, error) {
+					return "02/04/2023", nil
+				},
+			},
+			format:   "%m/%d/%Y",
+			expected: time.Date(2023, 2, 4, 0, 0, 0, 0, time.Local),
+		},
+		{
+			name: "simple short form with long year and slashes",
+			time: &ottl.StandardStringGetter[interface{}]{
+				Getter: func(ctx context.Context, tCtx interface{}) (interface{}, error) {
+					return "02/12/2022", nil
+				},
+			},
+			format:   "%m/%d/%Y",
+			expected: time.Date(2022, 2, 12, 0, 0, 0, 0, time.Local),
+		},
```

Comment:
Are these 2 tests the same?

Reply:
Hm, yeah this must have happened while I was changing tests around. The duplicate is removed.